### ISSUE TITLE
LeagueEvent has no field named 'byo_time' Error while loading initial data.

### ISSUE THREAD

**Issue by lmhsjackson** [climu/openstudyroom#302]
Following the initial setup in the wiki (I am in dev branch, but actually I want to try to start using this for our local leagues, should I be on master?)
It seems simple but I am not well versed in python. Any hints?

```
› ./manage.py loaddata fixtures/initial_data.json
Traceback (most recent call last):
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/db/models/options.py", line 566, in get_field
    return self.fields_map[field_name]
KeyError: 'byo_time'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/serializers/json.py", line 69, in Deserializer
    yield from PythonDeserializer(objects, **options)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/serializers/python.py", line 116, in Deserializer
    field = Model._meta.get_field(field_name)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/db/models/options.py", line 568, in get_field
    raise FieldDoesNotExist("%s has no field named '%s'" % (self.object_name, field_name))
django.core.exceptions.FieldDoesNotExist: LeagueEvent has no field named 'byo_time'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 365, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/management/base.py", line 335, in execute
    output = self.handle(*args, **options)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/management/commands/loaddata.py", line 72, in handle
    self.loaddata(fixture_labels)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/management/commands/loaddata.py", line 113, in loaddata
    self.load_label(fixture_label)
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/management/commands/loaddata.py", line 168, in load_label
    for obj in objects:
  File "/home/jackson/projects/openstudyroom/venv/lib/python3.5/site-packages/django/core/serializers/json.py", line 73, in Deserializer
    raise DeserializationError() from exc
django.core.serializers.base.DeserializationError: Problem installing fixture '/home/jackson/projects/openstudyroom/fixtures/initial_data.json': 
(venv) 

```

**Comment by climu** [comment]
Thanks for raising that.

I made some changes in the models to support goquest SGFs and didn't update the initials datas. To be specific I renamed the field `byo_time` to `additional_time` in this [commit](https://github.com/climu/openstudyroom/commit/bda3483a7c458097a73d9baf8aef4eb7d7415b97).

I will update initials datas today.

> (I am in dev branch, but actually I want to try to start using this for our local leagues, should I be on master

 I would suggest you start on dev with testing datas (folowing the wiki) to see how things work. Then if you want to deploy and use it for your project It depends on what you want to do (on a server behind nginx/gunicorn? locally on a go club computer?). Anyway, it's not the same issue. Feel free to ring me on OSR discord or to start issue here explaining what's your plan so we can help you deploy and use that.
